cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/Content/ProfileAssociations/index.js

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {MySearchApi} from 'api/search';
+import {isNilOrEmpty, humanReadableDuration} from 'services/helpers';
+
+require('./ProfileAssociations.scss');
+const HEADERS = [
+  {
+    label: 'Name',
+    property: 'name'
+  },
+  {
+    label: 'Namespace',
+    property: 'namespace'
+  },
+  {
+    label: 'Created',
+    property: 'created'
+  },
+  {
+    label: 'Last 24 hrs runs'
+  },
+  {
+    label: 'Total runs'
+  },
+  {
+    label: 'Last run node/hr'
+  },
+  {
+    label: 'Total node/hr'
+  },
+  {
+    label: 'Schedules'
+  },
+  {
+    label: 'Triggers'
+  }
+];
+
+export default class ProfileAssociations extends Component {
+  static propTypes = {
+    profile: PropTypes.object.isRequired,
+    namespace: PropTypes.string.isRequired
+  };
+
+  state = {
+    associationsMap: {}
+  };
+
+  componentDidMount() {
+    let {namespace, profile} = this.props;
+    MySearchApi
+      .search({
+        namespace: namespace,
+        query: `profile:${namespace}.${profile.name}`
+      })
+      .subscribe(
+        res => {
+          this.setState({
+            associationsMap: this.convertMetadataToAssociations(res.results)
+          });
+        },
+        err => {
+          console.log(err);
+        }
+      );
+  }
+
+  convertMetadataToAssociations = (metadata) => {
+    let appsMap = {};
+    metadata.forEach(m => {
+      let existingEntry = appsMap[m.entityId.application];
+      if (!(m.entityId.application in appsMap)) {
+        existingEntry = {
+          name: m.entityId.application,
+          namespace: m.entityId.namespace
+        };
+        appsMap[m.entityId.application] = existingEntry;
+      }
+      if (m.entityId.schedule) {
+        appsMap[m.entityId.application] = {
+          ...existingEntry,
+          schedules: [...(existingEntry.schedule || []), m.entityId]
+        };
+      } else if (!isNilOrEmpty(m.entityId.type)) {
```

Comment:
why the `else if` here? Can't both the `schedules` and `created` properties be set at the same time?

Reply:
Its not available in schedule.